packages/core/src/app/payment/hostedCreditCard/getHostedCreditCardValidationSchema.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { HostedCreditCardFieldsetValues } from '@bigcommerce/checkout/payment-integration-api';
+import HostedCreditCardFieldsetValues from '../paymentMethod/HostedCreditCardFieldsetValues'
```

Comment:
Is this intentional?

Reply:
Yes it is intentional, as we don't have the `HostedCreditCardFieldsetValues` in that package